app/src/main/database/index.ts

```diff
@@ -214,6 +218,16 @@ export class DB {
       SELECT uuid, data FROM journalists
     `);
 
+    this.selectUnseenItemsBySource = this.db.prepare(`
```

Comment:
maybe `selectPendingUnseenItemsBySource` instead? 

also, since we're specifying that it's unseen I think we can specify the type rather than passing it in:
```
...
AND pending_events.type = ${PendingEventType.Seen}
...
```